### ISSUE TITLE
Berserker Gauntlet Proc (hitall & twitch) strikes mounts

### ISSUE THREAD

**Issue by beshaba-toril** [torilmud/issues#111]
Fix coding that prevents the need to remove berserker gaunts in certain areas (i.e. - ChP, When using a mount, any zones with friendly mobs)

As the nasty spikes on your gauntlets tear into the flesh of your opponent
something inside your head triggers a seemingly infinite loop of carnage,
repeating itself over and over. Your movement speed increases with every loop
as you focus in on the bloodshed.
Your longsword glows dark and bites into a vicious nightmare's neck.
A vicious nightmare shivers slightly.
A vicious nightmare seems to be blinded!
Your massive slash slightly wounds an Ashstone noble.
Your massive slash slightly wounds an Ashstone noble.
Your massive slash slightly wounds an Ashstone noble.
The world starts spinning, and your ears are ringing!
You are too stunned to do that!

**Comment by shevarash** [comment]
I don't quite understand the issue. Can you give a little more detail?

**Comment by Inames** [comment]
@shevarash 

TL:DR make the gauntlets' proc's ignore player controlled mobs/pets/mounts and mobs that are otherwise un-targetable/friendly like the ChP guards. They are also capable of hitting/engaging mobs that you can't target with a normal kill command, like shopkeepers(as far as I recall). So that needs to be addressed as well.

Berserker gauntlets' procs that periodically hitall, and the the one that causes you to randomly attack a single mob in the room, can hit friendly pets/mounts/npc's that are normally not hit by friendly spells/hitall unless directly attacked first. The pets/mounts example is readily apparent. But the ChP example specifically refers to the mephit guards, and other mobs like them, where we can cast AoE spells without hitting them, even though they are not grouped with us, and are otherwise attackable mobs (can type kill guard, or backstab them, or target them with spells). The problem, is not really a problem thematically with the item, but mechanically where we have to remove the gauntlets to prevent causing problems. Berserker gauntlets often lead to blinding/killing/etc 'friendly' mobs/mounts/pets which can be rather irritating; such as in Beshaba's log segment above, where the player's gauntlets caused him to hit another player's nightmare and blinded it, which, due to entering/exiting indoor rooms, whilst dismounting to enter/leave, the player's nightmare can no longer follow, disrupting the flow of play.

* Very few 'friendly' mobs are like the ChP mephits, in fact I cannot readily think of any other mobs who are exempt from AoE spell targeting besides them. Most just get hit anyways.

I think the idea is that non-aggressive mobs shouldn't be targetable or strikable by some effects without designating that you want to hit them. Such as the hitall skill only striking at mobs who are aggressive to the user, so should *some* spells/skills/effects operate in a similar fashion.
